index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>About Me</title>
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1>About Me</h1>
+    <h2>A Little About Me</h2>
+    <p>My name is Sean Reigle and I'm a 33 year old aspiring software developer! I'm married and we currently live in the Lakeland Hills area of Auburn. Before deciding to pursue a career in technology, I was in the Army for a few years as an airborne infantryman. Needless to say, there isn't much opportunity for learning tech skills while jumping out of perfectly good airplanes and blowing stuff up for fun. After the Army, I decided to try my hand in the technology field having always had a passion for computers since I was messing around on Apple II computers playing The Oregon Trail. I found an opportunity at Expedia and siezed it. The work at Expedia led me to software development. I worked side by side with the software development team in charge of Expedia's implementation tools and loved how they created and implemented tools that the whole of Expedia as an organization was benefiting from. This is what led me to leave Expedia to pursue Code Fellows. </p>
+    <h2>My Background</h2>
+    <p>Before attending Code Fellows, I was an operations analyst at Expedia (exact boring job title was an International Operations and Traffic Analyst). During my time at Expedia, I learned quite a bit about the operations side of things, probably more than I ever wanted to. I realized after 10 months of working here that I wanted to be more DEV than OPs. I've always had a passion for creating things, and more specifically, creating things with computer languages...so, here I am! I cannot say that the Expedia experience was all for naught because I did pick up a few skill sets being there. The tools we used on an every day basis was mainly Splunk, GitHub, and Python created dashboards and monitoring</p>
+    <h2>My Education</h2>
+    <p>I received my undergraduate degree in Criminal Justice Administration from Columbia College and I'm pursuing my Bachelors in Information Technology Administrative Management with an emphasis in Cybersecurity from Central Washington University.</p>
+    <h2>My Goals</h2>
+    <p>My goals are many, but can be sorted into short and long term. My short term goals include being proficient enough in JavaScript, HTML, and CSS to be at 100% efficiency (or damn near) at my next gig. My long term goal is to land a career in software development.</p>
+    <script type="text/javascript" src="app.js"></script>
```

Comment:
i would put the white space between the `<p>` and the `<script>` tag instead of after the `<script>`